portfolio/src/main/webapp/index.html

```diff
@@ -1,65 +1,102 @@
+                        Some parts of the computer sciences that interest me the most are server-side development, cryptography and blockchain technology.<br>
+                     </h5>
+                     <br>
+                     <h4>Checkout my online profiles here :</h4>
+                     <br>
+                     <p>
+                        <a href="assets/resume.pdf" target="_blank"
+                           ><img src="https://img.icons8.com/windows/48/000000/resume.png" /></a>
+                        <a href="https://linkedin.com/in/na899" target="_blank"
+                           >
+                        <img src="https://img.icons8.com/windows/48/000000/linkedin.png" /></a>
+                        <a href="https://github.com/na899" target="_blank">
+                        <img src="https://img.icons8.com/windows/48/000000/github.png" /></a>
+                        <a href="https://medium.com/@pnaveena1999" target="_blank"> 
+                        <img src="https://img.icons8.com/windows/48/000000/medium-logo.png" /></a>
+                     </p>
+                  </div>
+               </div>
+               <div class="row">
+                  <div class="col-md-4"></div>
+                  <div class="col-md-4"> <img src="https://img.icons8.com/ios-filled/50/000000/down.png"/> <br>
+                     Keep scrolling to explore!
+                  </div>
+                  <div class="col-md-4"></div>
+               </div>
+            </div>
+         </section>
+         <section class="even  align-items-centers" id="two">
+            <h1>Terminal</h1>
+            <div id="terminal" >
+               Welcome to My website! Get to know me better! Type "help" to get started. <br>
+               <span>$</span> <input type='text' id = "choice"  >
+               <p id="output"></p>
+            </div>
+         </section>
+         <section class="odd  align-items-centers" id="three">
+            <div>
+               <h1>If you have any comments about this webpage please enter it here:</h1>
+               <div class="container">
+                  <form id="comment-form" class="form" >
+                     <div class=" row form-group">
+                        <label class="control-label col-sm-2" for="name-input">Name : </label>
+                        <div class="col-sm-10">
+                           <input type="text" id="name-input" class="form-control" name="name-input" placeholder="Name" required/>
+                        </div>
+                     </div>
+                     <div class=" row form-group">
+                        <label class="control-label col-sm-2" for="name-input">Comments : </label>         
+                        <div class="col-sm-10">
+                           <textarea name="comment-input" id="comment-input"  class="form-control" placeholder="Your Comments" required></textarea>

```

Comment:
Clear the form, after the comment has been posted. Currently the form stays as is